lambdas/evaluate-gpg45-scores/src/test/java/uk/gov/di/ipv/core/evaluategpg45scores/EvaluateGpg45ScoresHandlerTest.java

```diff
@@ -300,6 +272,9 @@ void shouldStoreRequiredScoresInSessionIfGpg45ProfileNotMatched() throws Excepti
 
         evaluateGpg45ScoresHandler.handleRequest(request, context);
 
+        ArgumentCaptor<IpvSessionItem> ipvSessionItemArgumentCaptor =
+                ArgumentCaptor.forClass(IpvSessionItem.class);
+        verify(ipvSessionService).updateIpvSession(ipvSessionItemArgumentCaptor.capture());
```

Comment:
You don't need to capture the session like this - I recently merged a change that made the session object a spy (`@Spy private IpvSessionItem ipvSessionItem;`).

You're not actually doing anything with that captured argument here either, so you could just call 
```
verify(ipvSessionService).updateIpvSession(ipvSessionItem);
```

Are you looking to ensure that the session was persisted after the required scores were added? If so, you might want to look at what I did recently after getting bitten and causing a P1. Need to check that the update happens *after* the write.  You could change the below to work for writing the required scores.

```
        InOrder inOrder = inOrder(ipvSessionItem, ipvSessionService);
        inOrder.verify(ipvSessionItem).setVot(VOT_P2);
        inOrder.verify(ipvSessionService).updateIpvSession(ipvSessionItem);
        inOrder.verify(ipvSessionItem, never()).setVot(any());
```

Reply:
Thanks comments. Done.